Managers/CardManager.gd

```diff
@@ -14,8 +17,25 @@ func set_card_container(in_card_container: CardContainer) -> void:
 	card_container = in_card_container
 	on_card_container_initialized.emit()
 	
+	if in_card_container != null:
+		_initialize_deck()
+
+
+func _initialize_deck() -> void:
+	# Set our default deck from the card container.
+	# NOTE: We assume that if the current_deck is empty, then this is the first time the game was loaded.
+	# So we load the default deck stored in the card container.
+	# TODO: It would be better to store default_deck in a separate Resource file
```

Comment:
indeed
and it's not really clear currently in the code where the default deck is (at least for me)

Reply:
The default deck is stored in the `CardContainer`, which doesn't make much sense any more. Should I go ahead and move it to a resource in this PR?

Reply:
Yes, it makes sense to do it now